config/config.go

```diff
@@ -292,7 +292,7 @@ var (
 		LogPath:                      "Log",
 		ChainDBPath:                  "ChainDB",
 		WalletFile:                   "wallet.json",
-		DefaultTlsDomainTmpl:         "{{.DashedIP}}.ipv4.nknlabs.io",
+		DefaultTlsDomainTmpl:         "{{.DashedIP}}.ipv4.{{.DomainName}}",
```

Comment:
We shouldn't assume domain is in the form of `*.ipv4.domain`. Simply change `DefaultTlsDomainTmpl` to `[]string` should be enough